api/proto/node/node.proto

```diff
@@ -153,3 +157,12 @@ message BatchHeader {
 	// The Ethereum block number at which the batch is dispersed.
 	uint32 reference_block_number = 3;
 }
+
+// Version info request
```

Comment:
It may be helpful to think a bit longer term. Beyond software version, maybe other node info will be desired, e.g. how many os, num CPUs etc. In that case, this API would be `GetNodeInfo`?